.eslintrc

```diff
@@ -54,11 +50,7 @@
     },
     {
       "files": ["**/*.ts", "**/*.tsx"],
-      "extends": [
-        "plugin:@typescript-eslint/eslint-recommended",
-        "plugin:@typescript-eslint/recommended",
-        "plugin:instawork/recommended-typescript"
-      ],
+      "extends": ["plugin:instawork/recommended-typescript"],
```

Comment:
any reasons to remove the two other extended plugins?

Reply:
Not really, these didn't exist in master so just to sync both.